test/VariablesTest.cs

```diff
@@ -11,15 +11,24 @@ public VariablesTest()
         }
 
         [Test]
-        public void TestValue()
+        public void TestGetValue()
         {
             Mocker.Mock(new Uri("test:///variables/foo%2Fbar/value"), "testvalue");
             Assert.AreEqual("testvalue", Client.Variable("foo/bar").GetValue());
 
             // TODO: not sure if this is supposed to be a plus or %20 or either
-            Mocker.Mock(new Uri("test:///variables/foo+bar/value"), "space test");
+            Mocker.Mock(new Uri("test:///variables/foo%20bar/value"), "space test");
```

Comment:
Looks like you checked and the plus doesn't work. if so, please remove the comment above.